test/test.py

```diff
@@ -77,7 +115,54 @@ def test_1_save(self):
         self.assertEqual(si32.precision, new_precision)
         os.unlink(new_filepath)
 
+        torchaudio.set_audio_backend(self.default_audio_backend)
+
+    def test_1_save_soundfile_sine(self):
```

Comment:
Doesn't the same advise as above apply where you don't need to copy paste the test code from the sox test

Reply:
It could also be interesting to try saving with sox, loading with sndfile and vice-versa.

Reply:
Good point. Added.